src/data/chargen/books/ninth-world-guidebook

```diff
@@ -0,0 +1,227 @@
+var descriptors = [
+  {
+    name: 'Coraoan',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
+    ],
+    stats: {
+      speed: 3
+    },
+    shins: 10,
+    skills: {
+      trained: [
+        'All tasks related to the history of Corao and the surrounding regions',
+        'All tasks having to do with the numenera'
+      ]
+    }
+  },
+  {
+    name: 'Desert-Dwelling',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
+    ],
+    stats: {
+      might: 4
+    },
+    shins: 10,
+    skills: {
+      trained: [
+        'Any task involving resisting damage from heat',
+        'All tasks having to do with finding food or water'
+      ],
+      inability: [
+        'Any task involving resisting damage from cold, and you have +2 to Armor that applies only to damage from heat',
+        'Any task involving swimming or handling watercraft'
+      ]
+    },
+    extras: [
+      'Long-suffering: You can go twice as long without food and water as another human.'
+    ]
+  },
+  {
+    name: 'Devout',
+    sourcebook: 'Ninth World Guidebook',
+    page: 216,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 216 }
+    ],
+    stats: {
+      intellect: 3
+    },
+    shins: 10,
+    extras: [
+      'Steadfast: Your faith gives you the foundation to deal with stress. When something would alter the difficulty of your action to your detriment, you can negate one step of the modification. Once you use this ability, you cannot use it again until after your next ten-hour recovery roll.'
```

Comment:
I'm concerned this exceeds the limits of what MCG allow for generators, as it contains basically the entire descriptive text of it. Stripping it back to a one sentence summary would probably be advised.